go-controller/pkg/libovsdbops/router.go

```diff
@@ -218,7 +218,7 @@ func DeleteLogicalRouterPorts(nbClient libovsdbclient.Client, router *nbdb.Logic
 	opModel := operationModel{
 		Model:            router,
 		OnModelMutations: []interface{}{&router.Ports},
-		ErrNotFound:      true,
+		ErrNotFound:      false,
```

Comment:
are these the same parts that @flavio-fernandes is touching here: https://github.com/ovn-org/ovn-kubernetes/pull/3647 ?

Reply:
@tssurya :  yes, definitely!  deleting what is not there should not fail.